gspread/spreadsheet.py

```diff
@@ -398,7 +425,7 @@ def del_worksheet(self, worksheet):
 
         return self.client.batch_update(self.id, body)
 
-    def del_worksheet_by_id(self, worksheet_id: Union[str, int]):
+    def del_worksheet_by_id(self, worksheet_id: Union[str, int]) -> Any:
```

Comment:
why not use Response type here?

Reply:
because the method `self.client.batch_update()` return a JSON value that we currently type as `Any`